python/ray/tune/tests/test_resource_updater.py

```diff
@@ -0,0 +1,41 @@
+import ray
+from ray.tests.conftest import *  # noqa
+from ray.tune.utils.resource_updater import ResourceUpdater
+
+
+def test_resource_updater(shutdown_only):
```

Comment:
Is this meant to be used?

Reply:
the test is included in BUILD, so it is used